contrib_bots/lib/myanimelist.py

```diff
@@ -0,0 +1,95 @@
+import requests, logging
+from xml.etree import ElementTree
+
+# Signup for a MAL account here https://myanimelist.net/register.php?from=%2F
+USERNAME = '[PLEASE CHANGE]'
+PASSWORD = '[PLEASE CHANGE]'
+
+# See myanimelist/docs.md for instructions on running this code.
+MAL_SEARCH_URL = "https://myanimelist.net/api/anime/search.xml"
+MAL_PAGE_URL = "https://myanimelist.net/anime/"
+
+# Checks if the username and password has been changed
+if USERNAME == "[PLEASE CHANGE]" or PASSWORD == "[PLEASE CHANGE]":
+    logging.error("Please update the USERNAME and PASSWORD on contrib_bots/lib/mal.py")
+    exit()
+
+class MyanimelistHandler(object):
+    '''
+    This plugin responds with the link of to the anime's
+    myanimelist page and some details along with
+    it in the same stream and topic
+    It looks for messages starting with '@mal'
+    '''
+    def search(self, query):
+        # Encode query into url
+        data = {
+            'q': query
+        }
+        try:
+            response = requests.get(MAL_SEARCH_URL, params=data, auth=(USERNAME, PASSWORD))
+        except requests.exceptions.ConnectionError:
+            logging.warning("Can not connect to myanimelist server.")
+        except:
+            logging.warning("An unknown error has occured when trying to get data.")
+        try:
+            xml = ElementTree.fromstring(response.content).find('entry')
+            content_dict = {
```

Comment:
So you're interested only in certain keys. How about you kept a list of those keys:
```
keys_of_interest = ['id' ...]
```
And used that list to get the value from the XML element